src/sentry/api/serializers/models/artifactbundle.py

```diff
@@ -0,0 +1,76 @@
+import base64
+
+from sentry.api.serializers import Serializer
+from sentry.models import ReleaseArtifactBundle, SourceFileType
+
+INVALID_SOURCE_FILE_TYPE = 0
+
+
+class ArtifactBundlesSerializer(Serializer):
+    def get_attrs(self, item_list, user):
+        release_artifact_bundles = ReleaseArtifactBundle.objects.filter(
+            artifact_bundle_id__in=[r.id for r in item_list]
+        )
+        release_artifact_bundles = {
+            release.artifact_bundle_id: (release.release_name, release.dist_name)
+            for release in release_artifact_bundles
+        }
+
+        return {
+            item: {
+                "bundle_id": item.bundle_id,
+                "release": release_artifact_bundles.get(item.id, (None, None))[0],
+                "dist": release_artifact_bundles.get(item.id, (None, None))[1],
+                "file_count": item.artifact_count,
+                "date": item.date_uploaded,
+            }
+            for item in item_list
+        }
+
+    def serialize(self, obj, attrs, user):
+        return {
+            "bundleId": str(attrs["bundle_id"]),
```

Comment:
am i missing something? why do we do it again manually? the purpose is to delegate it to serializer so no need to manually transform it to camelCase.

Reply:
Yeah this is also what I knew but looking at the codebase, we use the `serialize` method which accepts a `Serialize` instance that is custom in Sentry. The serializers are all in `src/api/serializers`.